vyper/utils.py

```diff
@@ -138,8 +138,6 @@ def is_varname_valid(varname):
         return False
     if varname.lower() in reserved_words:
         return False
-    if varname[0] == '~':
-        return False
```

Comment:
This wasn't possible? @DavidKnott said something about this before

Reply:
Yes, also discussed in pt1. Python ast prevents this. With tokenizer PR we can also consider completely removing the '~' operator now as well :smile:

Reply:
What does that operator do? I've never used it

Reply:
It's a boolean operator for complement.  Never used before myself either:
```
~ x
Returns the complement of x - the number you get by switching each 1 for a 0 and each 0 for a 1. This is the same as -x - 1.
```

Reply:
Really interesting :thinking: 
```
>>> a = 3
>>> "{0:b}".format(a)
'11'
>>> "{0:b}".format(~a)
'-100'
>>> ~a
-4
```